.travis.yml

```diff
@@ -27,10 +27,9 @@ script:
   - dotnet restore
   - dotnet build -c Release
   - dotnet test --no-build -c Release tests/Chiron.Tests/Chiron.Tests.fsproj
-  - dotnet pack --no-build -c Release src/Chiron
-  #- dotnet benchmarks\Chiron.Benchmarks\bin\Release\netcoreapp2.0\Chiron.Benchmarks.dll --class SwaggerSchema
+  - dotnet pack --no-build -c Release --include-symbols
```

Comment:
You don't need to generate a symbols nuget package. https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack?tabs=netcore2x